src/spaces/utilities.jl

```diff
@@ -153,6 +153,28 @@ end
 #######################################################################################
 # %% Walking
 #######################################################################################
+"""
+    normalize_position(pos, space::Union{AbstractGridSpace,ContinuousSpace})
```

Comment:
Given that in general we do not let users handle and access spaces directly, I think the top level and exported version of htis should be with `ABM` instead.